### PR TITLE
3.0-ify start/install-3-0

### PR DESCRIPTION
**Pull request by ghost**
https://p3-install-polymer-dot-polymer-project.appspot.com/3.0/start/install-3-0

### CODE REVIEW

File: app/3.0/start/install-3-0.md [hunk 1]
```diff
@@ -1,87 +1,107 @@
...
-    polymer init
-    ```
+[Download and run a Node.js installer for your operating system](https://nodejs.org/en/download/). This will install npm as well. 
 
-5. Select `polymer-2-application`.
+For more detailed instructions on installing npm and Node.js, see the [npm and Node.js install documentation](https://docs.npmjs.com/getting-started/installing-node).
 
-6. Serve your project.
+**Install a current or active LTS version of Node.js**. Polymer requires a current or active LTS version of Node.js. See our [Node.js support page](/{{{polymer_version_dir}}}/docs/tools/node-support) for more information.
+{ .alert}
```
Comment (arthurevans) [inline app/3.0/start/install-3-0.md]: Needs an alert type to display correctly. Should probably be `{.alert .alert-info}`

File: app/3.0/start/install-3-0.md [hunk 2]
```diff
@@ -1,87 +1,107 @@
...
+### Install Yarn {#yarn}
 
-    The Polymer CLI requires Node.js and npm as well as Bower. For full installation instructions, see [the Polymer CLI documentation](../docs/tools/polymer-cli).
+For Mac and Linux users, an easy way to install Yarn is to run the Yarn install script:
 
-    ```bash
-    npm install -g polymer-cli
-    ```
+```bash
+curl -o- -L https://yarnpkg.com/install.sh | bash
```
Comment (arthurevans) [inline app/3.0/start/install-3-0.md]: I have grave doubts about installing anything this way. Not that npm is a lot safer, but I'd be inclined to tell people to npm install this. 

/cc @justinfagnani @kevinpschaaf

File: app/3.0/start/install-3-0.md [hunk 3]
```diff
@@ -1,87 +1,107 @@
...
 
-    ```bash
-    npm install -g bower
-    ```
+```bash
+node --version
+```
 
-2. Install the Polymer CLI
+### Install Yarn {#yarn}
```
Comment (arthurevans) [inline app/3.0/start/install-3-0.md]: Sorry for my ignorance here, but I'm confused about whether we're still recommending yarn. 

/cc @justinfagnani @kevinpschaaf

**Comment by arthurevans** [comment]
Some questions for Kevin and Justin, otherwise LGTM. Reassigning to Kevin.

**Review by arthurevans — APPROVED** [review]

File: app/3.0/start/install-3-0.md (hunk 2 above)
Comment (katejeffreys) [reply app/3.0/start/install-3-0.md]: meh I just copied what it says in the Yarn docs.. I should probably just link to them

File: app/3.0/start/install-3-0.md (hunk 3 above)
Comment (katejeffreys) [reply app/3.0/start/install-3-0.md]: me too

**Comment by katejeffreys** [comment]
- [ ] remove refs to yarn
- [ ] alert info thing
- [ ] proofread

File: app/3.0/start/install-3-0.md (hunk 2 above)
Comment (katejeffreys) [reply app/3.0/start/install-3-0.md]: doesn't matter since removing yarn

File: app/3.0/start/install-3-0.md (hunk 3 above)
Comment (katejeffreys) [reply app/3.0/start/install-3-0.md]: no yarn

**Pull request merged**